gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -136,15 +135,6 @@ trait GlutenTestsTrait extends GlutenTestsCommonTrait {
 
     if (canConvertToDataFrame(inputRow)) {
       glutenCheckExpression(expr, expected, inputRow)
-    } else {
```

Comment:
Could you add a comment to "checkEvaluation" to indicate that the function only verifies the native result and skips the vanilla check.

Reply:
Maybe, it would be better to add a log to indicate the evaluation is skipped.